entry.go

```diff
@@ -223,6 +250,44 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 	return nil
 }
 
+func (te *TunaEntry) nanoPayClaim(stream *smux.Stream, npc *nkn.NanoPayClaimer, lastComputed, lastClaimed *common.Fixed64, lastUpdate *time.Time) {
```

Comment:
This function is still a large chunk of duplicate code compared to exit. It needs to be combined.